fleetctl/list_unit_files.go

```diff
@@ -68,7 +68,45 @@ var (
 		"target":   mapTargetField,
 		"tmachine": mapTargetField,
 		"state": func(u schema.Unit, full bool) string {
-			if suToGlobal(u) || u.CurrentState == "" {
+			if suToGlobal(u) {
+				states, err := cAPI.UnitStates()
```

Comment:
This is pretty wasteful. Could we pass in the states through the map function instead?